packages/boxel/tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitReturns": true,
-    "noEmitOnError": true,
+    "noEmitOnError": false,
```

Comment:
This is necessary because ember-cli-typescript is trying to typecheck the app with tsc, which won't work because it doesn't understand gts files. Instead we are separately type checking the app with glint, which covers both ts and gts. This also means you see type errors in the console, although they are not really relevant.

We can fix this once https://github.com/embroider-build/embroider/pull/1236 releases and we upgrade to that embroider version. With that, you can remove ember-cli-typescript while still keeping typescript transpiration support.